src/main.c

```diff
@@ -483,6 +489,16 @@ static void set_shared_uss_env(ConfigManager *configmgr) {
   shared_uss_env = malloc(maxRecords * sizeof(char*));
   memset(shared_uss_env, 0, maxRecords * sizeof(char*));
 
+  char *configEnv = malloc(PATH_MAX+32);
+  char *runtimeEnv = malloc(PATH_MAX+32);
+  char *haEnv = malloc(256);
+  snprintf(configEnv, PATH_MAX+32, "ZWE_CLI_PARAMETER_CONFIG=%s", zl_context.config_path);
+  shared_uss_env[idx++] = configEnv;
+  snprintf(runtimeEnv, PATH_MAX+32, "ZWE_zowe_runtimeDirectory=%s", zl_context.root_dir);
+  shared_uss_env[idx++] = runtimeEnv;
+  snprintf(haEnv, 256, "ZWE_CLI_PARAMETER_HA_INSTANCE=%s", zl_context.ha_instance_id);
```

Comment:
for the record, the reason why we set these at all (and how its possible to bypass zwe) is because the configmgr zwe ts code looks for them specifically at startup, and derives everything else from them.

https://github.com/zowe/zowe-install-packaging/blob/a2a12b134999ebb490550ef3fc02bcfd3a0e760d/bin/libs/configmgr.ts#L33-L47


https://github.com/zowe/zowe-install-packaging/blob/a2a12b134999ebb490550ef3fc02bcfd3a0e760d/bin/libs/config.ts#L281